proposals/NNNN-widget-support.md

```diff
@@ -0,0 +1,248 @@
+- how many widgets should be displayed (example: limited to 6 or indefinite using a scrollable area) 
+- and how to order them (alphabetically, chronogically or based on recent app usage).
+
+## Potential downsides
+
+The current design of SDL does not clearly separate displays and screens. Together with multi-display support there will be a proposal to redesign display and screen capabilities. This proposal should still be valid as it focuses on multi screen only.
+
+## Impact on existing code
+
+On SDL there are additional RPCs that need to be added. The HMI needs to design templates for widgets. This proposal did not focus on the mobile changes. For mobile libraries it's necessary to add the new RPCs and add support for multiple screen managers. This will also be part of a separate proposal.
+
+## Alternatives considered
+
+There are no alternatives considered. App services can be used to define additional widgets/app cards for certain service types such as weather or media.
+
+Below are alternatives to parts of this proposal:
+
+#### Screen related `OnHMIStatus`
+
+In the proposal widget screens should not follow HMI levels. RPCs sent to a widget screen would follow app policies of HMI_LIMITED. This alternative proposes OnHMIStatus for any screen instance.
+
+```xml
+<function name="OnHMIStatus" functionID="OnHMIStatusID" messagetype="notification" since="1.0">
+  :
+  <param name="screenID" type="Integer" mandatory="false" since="5.1" />
+</function>    
+```
+
+The additional parameter can used by the system to tell the app the HMI level of any screen instance. The parameter should be omitted for the predefined default screen for better backward compatibility. For other screens the screen ID specified by the app should be set in this parameter.
+
+By default widgets created are in HMI_BACKGROUND. This allows the app to fill the widget with content after it's created. If they get visible on the display the HMI should notify the app that the widget is now in HMI_FULL. This can be the case if the driver selects the widget area (e.g. the home screen shows app widgets). As widgets are not launchable the should automatically transit to HMI_FULL once they are visible.
+
+To limit the changes in the RPC and support backward compatibility audioStreamingState should be provided to all screens of one app. The HMI and SDL core must make sure to not send different audio states per app.
+
+System context can be screen related. A widget can be fully visible (system context MAIN) while the app's default screen can present the menu (system context MENU).
+
+#### Screen duplication by SDL libraries
+
+To reduce complexity on the head unit screen duplication can be supported by the SDL libraries. This could allow more flexibility to enable or display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen update (duplication not synchronized).
```

Comment:
Update to:
To reduce complexity on the head unit **screen, duplication** can be supported by the SDL libraries. This could allow more flexibility to enable or display duplication for existing screens. However this increases the number of RPCs to be sent by the app and the user might see a delay in screen update (duplication not synchronized).